src/Record.ts

```diff
@@ -12,7 +13,7 @@ export class ReadableRecord {
   public readonly time: bigint;
   public readonly size: bigint;
   public readonly last: boolean;
-  public readonly stream: Stream;
+  public readonly stream: Stream | ArrayBuffer;
```

Comment:
@atimin good point! I have added another private field for ArrayBuffer. The twist is to give a dummy stream to the stream field in the browser to avoid having an optional stream field that would break compatibility. Let me know if the new update makes sense.